config/features/deprecated_flags.go

```diff
@@ -32,6 +32,12 @@ var (
 		Usage:  deprecatedUsage,
 		Hidden: true,
 	}
+
+	deprecatedRegistrationCache = &cli.BoolFlag{
```

Comment:
Should this not be called `deprecatedEnableRegistrationCache` just to stick with the naming convention